### ISSUE TITLE
Expired payment in channel?

### ISSUE THREAD

**Issue by iljah** [ACINQ/eclair#752]
Sent 4 ln transactions through acinq node using github.com/btcontract/lnwallet, 3 of them expired and got stuck, one failed and was refunded after several h. Was able to save only one error message for expired and stuck transactions, attached.

![acinq](https://user-images.githubusercontent.com/2098356/48394884-e6613c00-e71d-11e8-873a-b96f07ec8f4a.png)

**Comment by iljah** [comment]
Expired payment hash: 81badbf4c48e53cfba4e63cd0e614e8d4d7255af98ff7376c5d6792e66099133

**Comment by pm47** [comment]
Your payment `81badbf4c48e53cfba4e63cd0e614e8d4d7255af98ff7376c5d6792e66099133` was forwarded to node `0279c22ed7a068d10dc1a38ae66d2d6461e269226c60258c021b1ddcdfe4b00bc4` which never fulfilled/failed it, so the payment timed out and we had to close the channel to that node.

We should then have immediately failed the incoming payment from your node, but due to a bug in #659 the command was ignored. We actually fixed the bug independently, and the htlc was finally failed on 2018-11-13 19:51:07,204, a few hours after you opened this issue, and the channel is now cleaned.

I think we can close this?
